ThirdPartyNotices.txt

```diff
@@ -11,14 +11,22 @@ bring it to our attention. Post an issue or email us:
 
 The attached notices are provided for information only.
 
-1. bazelbuild/bazel (https://github.com/bazelbuild/bazel/)
-2. google/desugar (https://android.googlesource.com/platform/external/desugar/+/master/)
-3. nunit/nunitlite (https://github.com/nunit/nunitlite/)
-4. zLibDll/minizip (http://www.winimage.com/zLibDll/minizip.html)
 
+1.  bazelbuild/bazel (https://github.com/bazelbuild/bazel/)
+2.  google/desugar (https://android.googlesource.com/platform/external/desugar/+/master/)
+3.  mono/boringssl (https://github.com/mono/boringssl)
```

Comment:
These changes imply that we're doing the wrong thing for the *repo* `ThirdPartyNotices.txt` file.  The purpose of the *repo*-level `ThirdPartyNotices.txt` are for third party dependencies *copied into this repo*, *not* from submodules or NuGet packages or the like.

`boringssl` is included via submodule.  `linker` is included via submodule.  etc.  These do not need to be present within *this* `ThirdPartyNotices.txt` file.

Third party code such as this *does* need to be listed in the generated and included-within-the-installer `bin/$(Configuration)/lib/xamarin.android/ThirdPartyNotices.txt` file.